src/components/Card.js

```diff
@@ -1,40 +1,85 @@
 //"Components" has been renamed to "components in github"
 //my computer throws an error when I import in "index.js"
 export default class Card {
-  constructor(data, cardSelector, handleImageClick) {
-    this.name = data.name;
-    this.link = data.link;
+  constructor(
+    { isLiked, name, link, _id },
+    cardSelector,
+    handleImageClick,
+    openDeleteForm,
+    addLikeButton,
+    removeLikeButton
+  ) {
+    this.name = name;
+    this.link = link;
+    this._isLiked = isLiked;
+    this.id = _id;
     this.cardSelector = cardSelector;
     this._handleImageClick = handleImageClick;
+    this._addLikeButton = addLikeButton;
+    this._removeLikeButton = removeLikeButton;
+    this.openDeleteForm = openDeleteForm;
+    this._popupDeleteForm = document.querySelector("#delete-card-modal");
```

Comment:
`    this._popupDeleteForm = document.querySelector("#delete-card-modal");
`


Please, delete it because you don't use it

Reply:
Erased